livekit-ffi/src/conversion/room.rs

```diff
@@ -24,8 +26,7 @@ use livekit::{
         prelude::{ContinualGatheringPolicy, IceServer, IceTransportsType, RtcConfiguration},
     },
 };
-
-use crate::{proto, server::room::FfiRoom};
+use proto::ChatMessage;
```

Comment:
can you use proto:: instead (not importing ChatMessage). 
So the code can be clearer (nit tho)